tools/test-rules-writer/test_rules_writer.go

```diff
@@ -5,26 +5,23 @@ import (
 	"fmt"
 	"os"
 
-	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
-
 	"kubevirt.io/ssp-operator/pkg/monitoring/rules"
 )
 
 func main() {
-	const runbookTemplate = "test-runbook:%s"
-
-	allRules := append(rules.RecordRules(), rules.AlertRules(runbookTemplate)...)
```

Comment:
When I was writing the unit tests, I found out that the order of rule definition mattered. Record rules needed to be defined before any alert rule that used them. 
With this change, the record and alert rules are split into 2 groups. Could maybe the test be flaky, because it evaluates the 2 groups in random order?

Can you please try to fix the flakiness before we merge this PR?

Reply:
fixed thanks a lot!